test/test.js

```diff
@@ -14,13 +14,14 @@ var benchmarkLibs = fs.readdirSync('benchmarks').filter(function (script) {
 describe('main test suite', function () {
   it('test sourcemaps', function () {
     var res = optimizeJs('var baz = function () { console.log("foo") }()', {
-      sourceMap: true
+      file: 'optimized.js',
+      source: 'original.js',
+      sourceMap: true,
+      includeContent: true
```

Comment:
Will do, but I'm currently trying to get it to work with jquery, it currently did come close to working with ember.

The issue is this needs to be done after minify and a minified map is quite complicated.  I'll push up a demo of the issue.